### PR TITLE
Removed Kamon metrics from the code base

### PR DESCRIPTION
**Pull request by zen-dog**
Jira: MARATHON-8459

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## Changes to 1.8.xxx
+
+### Removed deprecated metrics
+We removed deprecated Kamon based metrics from the code base (see the 1.7.xxx changelog for details on new metrics). This lead to removal of deprecated command line arguments e.g. old reporter like `--reporter_graphite`, `--reporter_datadog`, `--reporter_datadog` and `--metrics_averaging_window`.
```
Comment (ichernetsky) [inline changelog.md]: `This leads`

File: src/main/scala/mesosphere/marathon/DeprecatedFeatures.scala [hunk 2]
```diff
@@ -47,13 +47,7 @@ object DeprecatedFeatures {
     softRemoveVersion = SemVer(1, 7, 0),
     hardRemoveVersion = SemVer(1, 8, 0))
 
-  val kamonMetrics = DeprecatedFeature(
```
Comment (ichernetsky) [inline src/main/scala/mesosphere/marathon/DeprecatedFeatures.scala]: Shouldn't it be kept, so that users get more descriptive error messages?

Comment (zen-dog) [reply src/main/scala/mesosphere/marathon/DeprecatedFeatures.scala]: Good catch. Reverted

**Comment by dpsoft** [comment]
@timcharper may I ask what is the motivation to go from `kamon` to `metrics`?

**Pull request merged**